index.html

```diff
@@ -257,12 +276,21 @@
         <a title="user agent">User agents</a> MUST NOT provide Push API access to <a title=
         "webapp">webapps</a> without the <a>express permission</a> of the user. <a title=
         "user agent">User agents</a> MUST acquire consent for permission through a user interface
-        for each call to the <code>subscribe()</code> method, unless a prearranged trust
-        relationship applies.
+        for each call to the <code>subscribe()</code> method, unless a previous permission grant
+        has been persisted, or a prearranged trust relationship applies. Permissions that are
```

Comment:
Wouldn't we consider previously granted permissions to be a prearranged trust relationship in context of the current session?